packages/sources/svelte/src/components/forms/VtmnTextInput/VtmnTextInput.svelte

```diff
@@ -94,6 +96,8 @@
   <div class="vtmn-text-input_container">
     <input
       bind:value
+      on:input
```

Comment:
Hi @ChrisHendrX thanks for your proposition.
In Svelte, it's not possible to spread all events? Instead of having to think about all the events that we need to forward... In React, we can thanks to {...props}
Thanks in advance.

Reply:
@ChrisHendrX, what's your opinion please regarding my comment just above? Thanks.